.travis.yml

```diff
@@ -1,4 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
+  - '10'
+  - '8'
+  - '6'
```

Comment:
please add 0.10, 0.12 and 4 to the build matrix

Reply:
@phated I've been able to run the tests on node 4.
But it fails on Node 0.12.18:

```
node_modules/mocha/bin/mocha:10
const spawn = require('child_process').spawn;
^^^^^
SyntaxError: Use of const in strict mode.
```
Is it possible to drop 0.10 & 0.12?

Reply:
reduce the mocha version to `^3` and it works. Mocha isn't a critical dep.

Reply:
@phated sorry for the delay.
Ok done & working.